codelabs/gke-base/install/setup.sh

```diff
@@ -92,8 +92,7 @@ gcloud beta pubsub subscriptions create $GCR_SUB --topic $GCR_TOPIC
 bold "Creating your cluster $GKE_CLUSTER..."
 
 gcloud container clusters create $GKE_CLUSTER --zone $ZONE \
-  --service-account $SA_EMAIL \
-  --username admin --cluster-version 1.8.8-gke.0 \
```

Comment:
What changed that we can omit this now? Update in gcloud behavior?

Reply:
It was possible before, but for the codelab this was copied from (the IO one) we needed a higher versioned cluster than the default.